src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -66,12 +61,15 @@ public UniquePersonList(Collection<Person> persons) throws DuplicatePersonExcept
     }
 
     /**
-     * Constructs a shallow copy of the list.
+     * Constructs a shallow copy of the list and sort.
      */
     public UniquePersonList(UniquePersonList source) {
+        UniquePersonList qpl = new UniquePersonList();
```

Comment:
This variable doesn't seem to do anything? 
Also, the naming is not very appropriate as it's rather difficult to tell what this is for.